bip-0077.mediawiki

```diff
@@ -0,0 +1,237 @@
+<pre>
+  BIP: 77
+  Layer: Applications
+  Title: Payjoin Version 2: Serverless Payjoin
+  Author: Dan Gould <d@ngould.dev>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0077
+  Status: Draft
+  Replaces: 78
```

Comment:
I don't think this really replaces BIP78 since it's intended to be backwards-compatible?